app/main/views/suppliers.py

```diff
@@ -444,3 +443,28 @@ def create_your_account_complete():
         "suppliers/create_your_account_complete.html",
         email_address=email_address
     ), 200
+
+
+@main.route('/<framework_slug>/opportunities-overview', methods=['GET'])
+@login_required
+def opportunities_overview(framework_slug):
```

Comment:
Yeah, going forward we'll be adding various links on this page and it was decided that splitting it out by framework would work with a view to going forward, adding frameworks, reusability etc.